Jenkinsfile

```diff
@@ -0,0 +1,79 @@
+@Library('fh-pipeline-library')
+import org.feedhenry.Utils
+
+def apbName = 'test-apb'
+def projectName = 'ci-apb-test'
+
```

Comment:
Could you add this stage here, before going into any `node` blocks, please? https://github.com/aerogear/mobile-cli/blob/master/Jenkinsfile#L6..L8

Also please add that to any future Jenkinsfiles that will be run on PRs 👍